### PR TITLE
Supported on Chrome 56 and Opera 43

### PR DESCRIPTION
**Pull request by laukstein**
Status https://www.chromestatus.com/feature/5680800376815616

### CODE REVIEW

**Comment by Elchi3** [comment]
This is https://www.chromestatus.com/feature/5639972996513792
(Chrome 56, Opera 43 are correct)

File: http/headers/referrer-policy.json [hunk 1]
```diff
@@ -5,11 +5,11 @@
         "support": false
       },
       "Chrome": {
-        "support": false,
+        "support": "56",
         "notes": ["See <a href=\"https://bugs.chromium.org/p/chromium/issues/detail?id=619228\">Chromium bug 619228</a>."]
```
Comment (Elchi3) [inline http/headers/referrer-policy.json]: Please remove this note, now that the feature is supported.

Comment (laukstein) [inline http/headers/referrer-policy.json]: Done as you asked, https://github.com/mdn/browser-compat-data/pull/88/commits/5071e11251d4764292bc686b1af115eeeb920722

**Comment by Elchi3** [comment]
Thank you!

**Pull request merged**